backend/src/main/resources/schema.sql

```diff
@@ -43,9 +51,11 @@ CREATE TABLE Invoices
     bankgiro     VARCHAR(16),
     seller       VARCHAR(128) REFERENCES Organizations (email),
     customer     JSONB NOT NULL,
+    invoice_items JSONB,
```

Comment:
probably want to rename this back to `items` to make sure Spring can auto convert this to invoice